### PR TITLE
Adds S3 storage configuration to Helm chart

### PR DESCRIPTION
**Pull request by briancharous**

### CODE REVIEW

File: docs/content/install/install-on-kubernetes.md [hunk 1]
```diff
@@ -141,6 +141,17 @@ To use the Mongo DB storage provider, you will first need a MongoDB instance. On
 helm install gomods/athens-proxy -n athens --namespace athens --set storage.type=mongo --set storage.mongo.url=<some-mongodb-connection-string>
 ```
 
+#### S3 Configuration
+
+To use S3 storage with, set `storage.type` to `s3` and set `storage.s3.region` and `storage.s3.bucket` to the desired AWS region and
```
Comment (marpio) [inline docs/content/install/install-on-kubernetes.md]: To use S3 storage with `Athens`

File: docs/content/install/install-on-kubernetes.md [hunk 2]
```diff
@@ -141,6 +141,17 @@ To use the Mongo DB storage provider, you will first need a MongoDB instance. On
...
+S3 bucket name, respectively. By default, Athens will attempt to load AWS credentials using the AWS SDK from a the chain of environment
```
Comment (marpio) [inline docs/content/install/install-on-kubernetes.md]: from `a the` chain of environment

File: docs/content/install/install-on-kubernetes.md (hunk 1 above)
Comment (briancharous) [reply docs/content/install/install-on-kubernetes.md]: Oops, thanks for proofreading!

**Comment by briancharous** [comment]
Thanks for the quick review @marpio, I've fixed those typos and bumped the version.

**Pull request merged**